packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -22,6 +23,15 @@ import {
   HostsInterface,
 } from '../types';
 
+const defaultHosts: Required<HostsInterface> = {
```

Comment:
This doesn't need to be marked `Required` I don't believe.

Reply:
this is just the defaults and we are supplying them, so we should be able to leave it as required

Reply:
Chatted with Brooke offline- this is temporary and another PR unifies these data structures that are used in multiple places. So this works for me for now, so long as we clean it up later.

Reply:
SGTM, resolving.